### DISCUSSION TITLE
HA Core con wireguard client

### DISCUSSION THREAD

**Discussion by robypana** [indomus/forum#2027] [HUB: Home Assistant]
Buongiorno a tutti,
grazie alle Vostre preziose guide, mi sto cimentando nell'adozione di HomeAssistant
Ho scelto l'installazione Core (al momento la sto testando in docker su Windows, in attesa che prezzi e disponibilità dei Raspberry tornino a livelli accettabili).
Ho anche attivato un micro server su GCP per l'accesso da remoto (ho la connessione internet sotto NAT) e volevo implementare wireguard come tunnel VPN.
Sul server in GCP nessun problema ma come installo il client wireguard su HA Core ?
Scusate se ho scritto cavolate ma sto cercando di capirci qualcosa.
Grazie a tutti
Roberto

**Comment by snakuzzo** [comment]
Su core non è che puoi installare wireguard :)

Devi tirare su un container con wireguard a parte (es. https://github.com/monstrenyatko/docker-wireguard-client)
